gitfile/gitfile.go

```diff
@@ -57,23 +52,33 @@ func (g *GitFile) GetFileContents(ctx context.Context, namespace, repoUrl, filep
 	}
 
 	request := g.client.R().SetContext(ctx).SetBearerAuthToken(headerStruct.Authorization)
-	var errMsg ErrorMessage
+	var errMsg errorMessage
 	resp, err := request.
 		SetError(&errMsg).
 		Get(fileUrl)
 	if err != nil {
-		zap.L().Error("Failed to make GitHub URL call", zap.Error(err))
-		return nil, fmt.Errorf("GitHub file call failed: %w", err)
+		zap.L().Error("Failed to make file content request", zap.Error(err))
+		return nil, InternalError{fmt.Sprintf("File content request failed: %s", err.Error()), err}
 	}
 	zap.L().Debug(fmt.Sprintf(
 		"GitHub file call response code: %d", resp.GetStatusCode()))
+	// 2xx
 	if resp.IsSuccess() {
 		return io.NopCloser(bytes.NewBuffer(resp.Bytes())), nil
 	}
+	// >= 400
 	if resp.IsError() {
-		return nil, fmt.Errorf("%w. Status code: %d. Message: %s", failedFileRequestError, resp.GetStatusCode(), errMsg.Message)
+		switch resp.StatusCode {
+		case 400:
+			return nil, InternalError{"File content request has wrong format", nil}
+		case 401, 402, 403, 404:
```

Comment:
I like to use go constants for http codes, like `http.StatusBadRequest` etc.

Reply:
ok